front/app/components/admin/PostManager/components/PostTable/Row/IdeaRow.tsx

```diff
@@ -360,12 +359,8 @@ const ideaSource = {
               const hasPhases = !isEmpty(idea.data.relationships.phases.data);
 
               if (hasPhases) {
-                const ideaTitle = props.localize(
-                  idea.data.attributes.title_multiloc
-                );
                 const message = props.intl.formatMessage(
-                  messages.losePhaseInfoConfirmation,
-                  { ideaTitle }
+                  messages.loseIdeaPhaseInfoConfirmation
                 );
```

Comment:
I couldn't easily solve the type error with localize here, so I checked why it's used: to display information that's not really necessary on a page that I suspect is not used much anyway. So I took ideaTitle out. So this branch actually requires (unimportant) translations.

Reply:
Imo no need to start branch translations for this, we can just merge with master and let translations happen there.